cmd/thanos/query.go

```diff
@@ -141,6 +144,17 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Errorf("Address %s is duplicated for --rule flag.", dup)
 		}
 
+		// Check if the YAML configuration of request.logging is correct. Exit early if error.
+		HTTPlogOpts, err := logging.DecideHTTPFlag(*reqLogDecision, reqLogConfig)
```

Comment:
```suggestion
		httpLogOpts, err := logging.DecideHTTPFlag(*reqLogDecision, reqLogConfig)
```

Abrev should be capital unless it's starting word and we want private variable (:

Reply:
Yeah, this makes sense